tests/de/gurkenlabs/litiengine/entities/CombatEntityTests.java

```diff
@@ -0,0 +1,82 @@
+package de.gurkenlabs.litiengine.entities;
+
+import de.gurkenlabs.litiengine.Game;
+import de.gurkenlabs.litiengine.abilities.Ability;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.any;
+
+public class CombatEntityTests {
+
+    private CombatEntity combatEntity;
+    private CombatEntityListener entityListener;
+    private Ability ability;
+
+    @BeforeEach
+    public void setUp(){
+        Game.init(Game.COMMADLINE_ARG_NOGUI);
+
+        combatEntity = new CombatEntity();
+
+        entityListener = mock(CombatEntityListener.class);
+
+        combatEntity.addCombatEntityListener(entityListener);
+
+        ability = mock(Ability.class);
+    }
+
+    @Test
+    public void testHit_notDead(){
```

Comment:
Please try to avoid underscores in method names.

Reply:
Fixed